### PR TITLE
Improved functionality of required parameter

### PR DESCRIPTION
**Pull request by seroy**
This PR implements proposal described in comments for #19.
Unfortunately, I not have enought free time right now for do more stuff like tests and documentation fixes, so it provided as is.

### CODE REVIEW

**Comment by unaizalakain** [comment]
Awesome! I can go through the commit and update the documentation later today.

File: localized_fields/fields/field.py [hunk 1]
```diff
@@ -32,6 +32,13 @@ def __init__(self, *args, **kwargs):
 
         super(LocalizedField, self).__init__(*args, **kwargs)
 
+        if self.required is None and self.blank:
+            self.required = []
+        elif self.required is None and not self.blank:
+            self.required = [settings.LANGUAGE_CODE]
+        elif type(self.required) == bool and self.required:
```
Comment (unaizalakain) [inline localized_fields/fields/field.py]: What about `elif self.required is True`?

File: localized_fields/fields/field.py [hunk 2]
```diff
@@ -199,36 +206,25 @@ def validate(self, value: LocalizedValue, *_):
...
 
         defaults = {
             'form_class': LocalizedFieldForm
         }
-
+        if issubclass(kwargs.get('form_class', LocalizedFieldForm),
+                      LocalizedFieldForm):
+            defaults.update({
+                'required_langs': self.required
+            })
```
Comment (unaizalakain) [inline localized_fields/fields/field.py]: Should we not require that the form field has to know how to deal with it?

File: localized_fields/forms.py [hunk 3]
```diff
@@ -19,16 +19,13 @@ class LocalizedFieldForm(forms.MultiValueField):
...
         fields = []
 
         for lang_code, _ in settings.LANGUAGES:
-            field_options = {'required': False}
-
-            if lang_code == settings.LANGUAGE_CODE:
-                field_options['required'] = kwargs.get('required', True)
+            field_options = {'required': lang_code in required_langs}
 
             field_options['label'] = lang_code
```
Comment (unaizalakain) [inline localized_fields/forms.py]: Nitpicking, but why not
```
field_options = {'required': lang_code in required_langs,
                         'label': lang_code}
```

**Comment by Photonios** [comment]
This is great work and you robbed me of the chance to have a crack at this this weekend! Instead, I'll go over this :-)

Thanks!

File: localized_fields/fields/field.py (hunk 2 above)
Comment (seroy) [inline localized_fields/fields/field.py]: This is a moot point, but right now we have `LocalizedAutoSlugField` which set `form_class` as `CharField`, and `CharField` is not expects `required_langs` parameter.

**Comment by Photonios** [comment]
I looked more closely at this PR now. It's good work and I'd like to merge it after the comments have been fixed and some tests have been added. I could add the tests myself if you're short on time @seroy. 

One thing I'd like to see changed, or maybe you have a good reason for doing so is the name `required_langs`. Can't we just name this `required` like in the field class?

**Comment by Photonios** [comment]
This is great! Many thanks @seroy. Sorry for hi-jacking your changes and applying mine on top, I was guessing you were busy as you mentioned in your initial PR :)

**Pull request merged**